azuredeploy.json

```diff
@@ -269,5 +272,11 @@
         "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanName'))]"
       }
     }
-  ]
+  ],
+  "outputs": {
+      "databaseServer": {
+          "type": "string",
+          "value": "[variables('databaseFqdn')]"
```

Comment:
I believe this will output a secret (the admin password) which will stick around in various logs and the Azure UI which is not desirable. Given that the DB FQDN isn't used in the current template workflow, do we really need to include it here?

Reply:
it doesn't output the the parts before the 2nd `@` -- just `"databaseFqdn" : "[concat( variables('databaseServerName'), '.postgres.database.azure.com:', variables('databaseServerPort'))]",`